apps/web/src/components/PostDialog/PostDialog.jsx

```diff
@@ -12,12 +12,15 @@ const PostDialog = ({
   const navigate = useNavigate()
   const location = useLocation()
 
+  // Add state to track the container's scroll position
```

Comment:
Whats all this for?

Reply:
sometimes the post dialog appears in the wrong place when the outer page is scrolled down. I saw this too though I actually want to resolve it by not using the shadcn/radix PostDialog anymore and going back to our own Modal. later though